CalendarApp/GoogleCalendarService/BookingService.cs

```diff
@@ -4,23 +4,34 @@
 using System;
 using System.Collections.Generic;
 using System.Linq;
+using GoogleCalendarService.Manager;
 
 namespace GoogleCalendarService
 {
     public class BookingService
     {
-        private CalendarService _service;
-        private ILog _log;
+        private readonly CalendarService _service;
+        private readonly ILog _log;
+        public EventManager EventManager;
+        public string TimeZone;
```

Comment:
This field is used in post/update controller,for converting event to the right state.